### ISSUE TITLE
[Skill] Knight of the Blazing Sun's Focused Offense tactic identified as curse/ailing

### ISSUE THREAD

**Issue by vove-RoR** [WarEmu/WarBugs#3825]
Yesterday I was in a party and strange thing occurred. One of the party members (Knight of the Blazing Sun) was permanently marked as cursed/ailed. The Rune of Cleansing was cast-able on him when targeted but I couldn't dispel it off him as in fact he wasn't affected by any curse or ailing.

Now I guess there must be something that either marks KotBS as cursed thus making the rune of cleansing active but not possible to cast as the effect cannot be removed. In the beginning I thought it's some of the items but when he got naked (pretty awkward to ask someone to do it :) ) I still had rune of cleansing active for him.

It also might be an error with Rune of Cleansing itself falsely identifying something as curse/ailing. I have seen that happen for the first time.

Here are two screenshots. On the first 1 you can see our party member at whom, when targeted, I cannot cast the Rune of Cleansing. On the 2nd one you can see that after targeting KotBS the Rune of Cleansing is active (doesn't remove anything though and doesn't trigger cooldown).

1.
![1](https://cloud.githubusercontent.com/assets/16596013/12390516/8cdf7e2e-bdd8-11e5-8ce9-da681b22ed9e.jpg)

2.
![2](https://cloud.githubusercontent.com/assets/16596013/12390527/98b2fcb2-bdd8-11e5-869e-64c19fcb8e83.jpg)

**Comment by vove-RoR** [comment]
Ok, we figured it out. It is trigged by the KotBS's tactic called Focused Offense. Once he turns it off the Rune of Cleansing is no longer active.

Kudoz to my guild mate Sotheryk who wanted to solve the mystery as much as I did.

**Comment by ozarmidrashim** [comment]
If true, this should affect all tanks.

**Comment by vove-RoR** [comment]
I chatted with some ironbreaker and a swordmaster today. In their case turning Focused Offense on doesn't make the Rune of Cleansing active.

I also checked different skill that changes one stats at the cost of the other: Divine Fury. It's a RP's tactic lowering healing done at the cost of damage done, it's also not recognized as a curse.

Seems it's only affecting the KotBS but to be honest so far I've checked it only with one KotBS.
